server/controllers/UsersController.js

```diff
@@ -0,0 +1,241 @@
+import Helpers from '../utils/helper';
+import { User } from '../models';
+
+/**
+ * @class UsersController
+ */
+class UsersController {
+  /**
+   * @description
+   * @param {any} req
+   * @param {any} res
+   * @returns {void}
+   * @memberof UsersController
+   */
+  static registerUser(req, res) {
+    User.find({
+      where: {
+        email: req.body.email
+      }
+    })
+      .then((user) => {
+        if (!user) {
+          return Helpers.createUser(req, res);
+        }
+        return res.status(401).send({
```

Comment:
Maybe status code should be 409 or just find a more suitable one https://developer.mozilla.org/en-US/docs/Web/HTTP/Status